monkey/infection_monkey/exploit/powershell_utils/auth_options.py

```diff
@@ -0,0 +1,10 @@
+from dataclasses import dataclass
+from typing import Union
+
+
+@dataclass
+class AuthOptions:
+    ip_addr: str
```

Comment:
I don't think the IP address is really part of the authentication options. It's information about the host.